target/linux/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -21,23 +21,26 @@ case "$FIRMWARE" in
 	zyxel,nbg7815)
 		caldata_extract "0:art" 0x1000 0x20000
 		;;
+	yuncore,ax880)
+		caldata_extract "0:ART" 0x1000 0x20000
+		;;
 	prpl,haze)
-                caldata_extract_mmc "0:ART" 0x1000 0x20000
+		caldata_extract_mmc "0:ART" 0x1000 0x20000
 		;;
 	esac
 	;;
 "ath11k/QCN9074/hw1.0/cal-pci-0000:01:00.0.bin"|\
 "ath11k/QCN9074/hw1.0/cal-pci-0001:01:00.0.bin")
 	case "$board" in
 	prpl,haze)
-                caldata_extract_mmc "0:ART" 0x26800 0x20000
+		caldata_extract_mmc "0:ART" 0x26800 0x20000
```

Comment:
Same here

Reply:
sorry for this. I thought i've replaced 4 spaces with tabs everywhere.

Reply:
Done.